component-models.json

```diff
@@ -312,12 +312,79 @@
         "label": "Image",
         "multi": false
       },
+      {
+        "component": "text-input",
+        "valueType": "string",
+        "name": "title",
+        "value": "",
+        "label": "Title"
+      },
+      {
+        "component": "select",
+        "name": "type",
+        "value": "h2",
+        "label": "Type",
+        "valueType": "string",
+        "options": [
+          {
+            "name": "h1",
```

Comment:
@mayur-satav We don't want to give ability to author H1 in cards, so can you take it out from options?